ecommerce-ui/src/App.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {shallow, render, mount} from 'enzyme';
+import App from './App';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<App />, div);
+  ReactDOM.unmountComponentAtNode(div);
+});
+
+it('shallow renders', () => {
+  const wrapper = shallow(<App />);
```

Comment:
Was really just looking for one snapshot test, so this works. Feel free to go back and fix the others when you have time.